drivers/can/can_mcp2515.c

```diff
@@ -216,7 +216,7 @@ static void mcp2515_convert_zcanframe_to_mcp2515frame(const struct zcan_frame
 
```

Comment:
The conversion in the other direction and possible to limit DLC here

Reply:
General reply here: https://github.com/zephyrproject-rtos/zephyr/pull/20218#discussion_r341544439